src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/VulnerabilitiesControl.xaml

```diff
@@ -0,0 +1,100 @@
+        x:Key="StringFormatConverter" />
+    </ResourceDictionary>
+  </UserControl.Resources>
+  <Grid
+      Margin="0,12,0,16">
+    <StackPanel
+        Orientation="Vertical">
+      <StackPanel
+          Orientation="Horizontal"
+          VerticalAlignment="Center">
+        <imaging:CrispImage
+          x:Name="_warningIcon"
+          Margin="0,0,5,0"
+          AutomationProperties.LabeledBy="{Binding ElementName=__vulnerabilityLabel}"
+          Moniker="{x:Static catalog:KnownMonikers.StatusWarning}"
+          ToolTip="{Binding ElementName=_vulnerabilityLabel, Path=Text}" />
+        <TextBlock
+          x:Name="_vulnerabilityLabel"
+          FontWeight="Bold">
+          <TextBlock.Text>
+            <MultiBinding Converter="{StaticResource StringFormatConverter}">
+              <Binding Source="{x:Static nuget:Resources.Label_Vulnerabilities}" />
+              <Binding Path="PackageVulnerabilityCount" />
+            </MultiBinding>
+          </TextBlock.Text>
+        </TextBlock>
+      </StackPanel>
+      <TextBlock
+        Margin="0,8,0,0"
+        TextWrapping="WrapWithOverflow"
+        AutomationProperties.Name="{x:Static nuget:Resources.Accessibility_VulnerabilitiesDescriptionName}">
+        <TextBlock.Text>
+          <MultiBinding Converter="{StaticResource StringFormatConverter}">
+            <Binding Source="{x:Static nuget:Resources.Label_VulnerabilitiesDescription}" />
+            <Binding Path="PackageVulnerabilityMaxSeverity" Converter="{StaticResource IntToVulnerabilitySeverityConverter}" />
+          </MultiBinding>
+        </TextBlock.Text>
+      </TextBlock>
+      <StackPanel
+        Margin="0,8,0,0"
+        Orientation="Vertical">
+        <TextBlock
+          Grid.Row="0"
+          Text="{x:Static nuget:Resources.Label_Details}"
+          FontWeight="Bold" />
+        <ItemsControl
+          ItemsSource="{Binding PackageVulnerabilities}">
+          <ItemsControl.ItemTemplate>
+            <DataTemplate>
+              <Grid>
+                <Grid.ColumnDefinitions>
+                  <ColumnDefinition Width="Auto" SharedSizeGroup="AdvisoryHeaderColumnGroup" />
+                  <ColumnDefinition Width="*" SharedSizeGroup="AdvisoryLinkColumnGroup" />
+                </Grid.ColumnDefinitions>
+                <TextBlock
+                  Grid.Column="0"
+                  Margin="0,1,8,1">
+                  <TextBlock.Text>
+                    <MultiBinding Converter="{StaticResource StringFormatConverter}">
+                      <Binding Source="{x:Static nuget:Resources.Label_VulnerabilityAdvisory}" />
+                      <Binding Path="Severity" Converter="{StaticResource IntToVulnerabilitySeverityConverter}" />
+                    </MultiBinding>
+                  </TextBlock.Text>
+                </TextBlock>
+                <TextBlock
+                  Grid.Column="1"
+                  Margin="8,1,0,1">
+                  <Hyperlink
```

Comment:
Add `ToolTip` to the link. It helps navigation and accessibility.

Reply:
Good call!

Reply:
Updated locally. Change will be reflected next time I commit and push.